integration-tests/originate-known-contracts.ts

```diff
@@ -16,19 +16,34 @@ CONFIGS().forEach(({ lib, setup, protocol }) => {
   let keyPkh: string = "";
   let keyInitialBalance: BigNumber = new BigNumber(0);
 
-  (async () => {
-    await setup(true);
-    fs.writeFile(`known-contracts-${protocol.substring(0,9)}.ts`, '', (err: any) => {
+  let protocolShort = protocol.substring(0, 9);
+  let outputFile = `known-contracts-${protocolShort}.ts`;
+  let writeOutput = (line: string): void => {
+    fs.writeFile(outputFile, line + '\n', (err: any) => {
+      if (err) {
+        console.error(err);
+      }
+    });
+  };
+  let appendOutput = (line: string): void => {
+    fs.appendFile(outputFile, line + '\n', (err: any) => {
       if (err) {
         console.error(err);
       }
     });
+  };
+
+  (async () => {
+    await setup(true);
+
+    writeOutput("import { KnownContracts } from './known-contracts';")
+    appendOutput("export const knownContracts" + protocolShort + "KnownContracts = {");
```

Comment:
I think `KnownContracts` should not be part of the string.

Reply:
Pushed a new version, which should fix this. The intention was to concatenate the type annotation, i.e. ` + ": KnownContracts"` but the colon got lost. Should be fixed now.